presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/NodeSelectionStats.java

```diff
@@ -28,6 +28,7 @@
 
     private final CounterStat bucketedPreferredNodeSelectedCount = new CounterStat();
     private final CounterStat bucketedNonPreferredNodeSelectedCount = new CounterStat();
+    private final CounterStat bucketedNonAliveNodeReplacedCount = new CounterStat();
```

Comment:
Nit: Since we are incrementing this when `node.getNodeStatus() == DEAD`, how about we just call them `bucketedDeadNodeReplacedCount` here?

Reply:
Rohit didn't want "dead" in the counter name. Hence for non-bucketed case, its called "preferredNonAliveNodeSkippedCount"